core/src/dird/ndmp_dma_storage.cc

```diff
@@ -669,7 +707,10 @@ drive_number_t NdmpGetNumDrives(UaContext* ua, StorageResource* store)
   /*
    * See if the mappings are already determined.
    */
-  if (!NdmpUpdateStorageMappings(ua, store)) { return drives; }
+  if (!NdmpUpdateStorageMappings(ua, store)) {
+    Dmsg0(200, "NdmpUpdateStorageMappings failed\n");
```

Comment:
Maybe the Dmsg should contain a prefix of "NdmpGetNumSlots:" and "NdmpGetNumDrives:" here and lin line 692

Reply:
Dmsg0 prints out __FILE__ and __LINE__